tests/serverpod_test_server/lib/src/models/empty_model/empty_model_relation_item.spy.yaml

```diff
@@ -0,0 +1,5 @@
+class: EmptyModelRelationItem
+table: empty_model_relation_item
+fields:
+  name: String
+  containerId: int, relation(parent=empty_model)
```

Comment:
Comment: Note that we have two relations in these two models.

Reply:
Good note! Will fix that 👍

Reply:
Done!